openapi/index.yaml

```diff
@@ -1,49 +1,265 @@
 swagger: "2.0"
 info:
-  version: 1.1.0
-  title: IO Function API
+  version: 1.0.0
+  title: IO Functions CGN operator search
   contact:
     name: IO team
     url: https://forum.italia.it/c/progetto-io
   x-logo:
     url: https://io.italia.it/assets/img/io-logo-blue.svg
   description: >
-    Documentation of the IO Function API here.
-host: api.cd.italia.it
-basePath: "/template"
+    Documentation of the IO operator search API here.
+host: search.cgn.italia.it
+basePath: "/api/v1"
 schemes:
   - https
 paths:
-  "/echo/{input}":
+  /online-merchants:
     parameters:
-      - name: input
+      - "$ref": "#/parameters/Name"
+      - "$ref": "#/parameters/ProductCategories"
```

Comment:
We should move `parameters` into each operation and repeat them, due to our codegen compatibility

Reply:
I've moved the parameter definition